seqsender.py

```diff
@@ -586,9 +574,9 @@ def start_submission(unique_name, config, upload_log_path, test, overwrite):
         submit_biosample_sra(unique_name, config, upload_log_path, test, "biosample", overwrite)
     elif config_dict["general"]["submit_SRA"] == True:
         submit_biosample_sra(unique_name, config, upload_log_path, test, "sra", overwrite)
-    elif config_dict["general"]["submit_Genbank"] == True:
+    if config_dict["general"]["submit_Genbank"] == True:
```

Comment:
**NOTE:** in our case submit_Genbank is _never_ true, bc we have turned it off until we can figure out batch submissions